rs/universal_canister/lib/src/lib.rs

```diff
@@ -7,17 +7,29 @@
 
 pub mod management;
 
-use hex_literal::hex;
 use ic_types::Cycles;
+use lazy_static::lazy_static;
 use universal_canister::Ops;
-/// The binary of the universal canister as compiled from
-/// `rs/universal_canister/impl`.
-///
-/// For steps on how to produce it, please see the README in
-/// `rs/universal_canister`.
-pub const UNIVERSAL_CANISTER_WASM: &[u8] = include_bytes!("universal-canister.wasm.gz");
-pub const UNIVERSAL_CANISTER_WASM_SHA256: [u8; 32] =
-    hex!("9c0b4ed1d729ffdd0e8d194df3be621f58a2fb86e1f1bbf556d89f43637de303");
+
+lazy_static! {
+    /// The WASM of the Universal Canister.
+    pub static ref UNIVERSAL_CANISTER_WASM: &'static [u8] = {
+        let vec = get_universal_canister_wasm();
+        Box::leak(vec.into_boxed_slice())
```

Comment:
Understanding question: is the `Box::leak` needed to keep the same type (`&[u8]`) as before to minimize the amount of changes? Maybe it would be still be better to expose a  simple`<Vec<u8>>`,  since the Rust [doc](https://doc.rust-lang.org/std/boxed/struct.Box.html#method.leak) on `Box::leak` explicitly mentions a memory leak
> Dropping the returned reference will cause a memory leak.

Reply:
Yes, it's to retain the type we had before which I also believe is a better type.

`Box::leak` indeed causes a memory "leak" but that's the whole point here. We want the memory to stay allocated for the whole lifetime of the process.

Reply:
OTOH I guess I also don't mind using `Vec<u8>` here. Let me see what changes are necessary for that ...

Reply:
Worth noting that `UNIVERSAL_CANISTER_WASM` seems to often (always?) be converted `to_vec()` when it's being installed

Reply:
That [was easy](https://github.com/dfinity/ic/pull/2502/commits/160471ea8837367c9155182a0dd156eb9163312d).